code/segmentedtp/pubsub.q

```diff
@@ -0,0 +1,106 @@
+// Pub/sub utilities for segmented tp process
+// Functionality for clients to subscribe to all tables or a subset
+// Includes option for subsrcibe to apply filters to received data
+
+/load schema from params, default to "sym.q"
+.proc.loadf[(src:$[`schemafile in key .proc.params;raze .proc.params`schemafile;"sym"]),".q"];
```

Comment:
I don't think this should be here. 
This pub/sub lib should be generic (not tickerplant specific) so it might well be that the TP requires a schema file to be set (in which case it should just exit if it doesn't exist, rather than defaulting)
The pub/sub lib though just requires there to be some tables to be defined when it is initialised.